### DISCUSSION TITLE
Any beautiful theme (commercial/free) availalble for Orchard Core?

### DISCUSSION THREAD

**Discussion by orchardc** [OrchardCMS/OrchardCore#7724] [Show and tell]
I'm looking for more beautiful themes for Orchard Core, either commercial or free.

Any suggestion is appreciated!!

**Comment by hishamco** [comment]
There are some themes for both site and admin theme OOB, also there are another theme in [OCC Themes](https://github.com/OrchardCoreContrib/OrchardCoreContrib.Themes), hope to add more themes in the upcoming future

**Reply by orchardc** [reply]
Thanks @hishamco it is good. but I'm still looking for any commerical template available for Orchard Core?

**Reply by hishamco** [reply]
I didn't hear about any commercial templates for OC, but if you need something special, please contact me in Gitter

**Comment by OrchardSkills** [comment]
Yes. Material Design Theme
https://youtu.be/KW3VjBS-Lhg
https://github.com/OrchardSkills/OrchardSkills.OrchardCore.MaterialDesignTheme
